models.py

```diff
@@ -51,13 +52,33 @@ class User(BaseModel):
     Relationship is not implemented yet as seen below.
  """
 class Note(db.Model):
+    __tablename__ = 'Note'
```

Comment:
I don't think this should have a tablename, as we won't have a "Note" table (we should either have one table for all Notes or separate ones for Posts, Comments and Replies)